tests/integration_tests/test_aws.py

```diff
@@ -92,10 +92,26 @@ def test_save_submission_file_s3(seeded_test_client, test_buckets):
 
     assert response["Body"].read() == filebytes
     assert metadata["submission_id"] == submission_id
-    assert metadata["filename"] == f"HS-{uuid}.xlsx"
+    assert metadata["filename"] == filename
     assert metadata["programme_name"] == "Leaky Cauldron regeneration"
 
 
+def test_save_submission_file_s3_ascii_safe(seeded_test_client, test_buckets):
+    filename = "example–file.xlsx"
+    ascii_safe_filename = "example-file.xlsx"
```

Comment:
With close inspection it's possible to see these are different, but it's not easy.

Maybe we can make this more obvious?

```suggestion
    filename = "example\u2013file.xlsx"
    ascii_safe_filename = "example-file.xlsx"
```

Reply:
Done!